### ISSUE TITLE
[MAS 3.3.2] [Labels & heading-Endpoint] Label is not asociated with the (+) Add button

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#2055]
Actual Result:
User will unable to understand the purpose of the button.

Expected Result:
Label should be defined for the (+) Add button available under the bot explorer. it should announce the associated label as "Endpoint add(+) button". 

Test Environment
Platform: Mac 10.15.3 Beta (19D49f)
App: Bot Framework Emulator V4
App version: Version 4.7.0-99577 (4.7.0-99577.99577)
Tool: Voice Over

Repro Steps:
1.Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to "Create new bot configuration" and create new bot.
4. navigate to End point(+)>> Bot explorer tool bar.
5. Observe the (+) button announcement.

User Impact:
User will unable to understand the purpose of the control (+) button, if label is not associated with the button, it will impact of efficiency of the screen reader dependent user.

Impacted MAS Link:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

Attachments
[Endpoint add button- DT issue.zip](https://github.com/microsoft/BotFramework-Emulator/files/4070983/Endpoint.add.button-.DT.issue.zip)

**Comment by v-rampul** [comment]
#HCL-BotFrameworkEmulator; #Emulator-MAC; #A11yMAS; #HCL; #MAS3.3.2; #A11ySev3
